### PR TITLE
#201: Create Sacramento profile and route users to Shelter homepage

### PR DESCRIPTION
**Pull request by jkondrat**

### CODE REVIEW

File: src/main/webapp/app/modules/login/login.tsx [hunk 1]
```diff
@@ -32,8 +34,11 @@ export class Login extends React.Component<ILoginProps, ILoginState> {
   };
 
   render() {
-    const { location, isAuthenticated } = this.props;
-    const { from } = location.state || { from: { pathname: '/', search: location.search } };
+    const { location, isAuthenticated, account } = this.props;
+    const defaultRoute = hasAnyAuthority(account.authorities, [AUTHORITIES.SACRAMENTO])
```
Comment (pmuchowski) [inline src/main/webapp/app/modules/login/login.tsx]: I think you should rename it, it's not default role, it's default path

File: src/main/java/org/benetech/servicenet/security/AuthoritiesConstants.java [hunk 2]
```diff
@@ -5,6 +5,8 @@
  */
 public final class AuthoritiesConstants {
 
+    public static final String SACRAMENTO = "SACRAMENTO";
```
Comment (pmuchowski) [inline src/main/java/org/benetech/servicenet/security/AuthoritiesConstants.java]: rest of the authorities have a ROLE_ prefix, I think it should have it too

File: src/main/resources/config/liquibase/changelog/20190606150430_insert_Sacramento_profile.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet id="20190606150430-1" author="jkondrat">
+        <insert tableName="jhi_authority">
+            <column name="name" type="varchar(50)">SACRAMENTO</column>
```
Comment (pmuchowski) [inline src/main/resources/config/liquibase/changelog/20190606150430_insert_Sacramento_profile.xml]: as above

File: src/main/webapp/app/config/constants.ts [hunk 4]
```diff
@@ -7,6 +7,7 @@ export default config;
 export const SERVER_API_URL = process.env.SERVER_API_URL;
 
 export const AUTHORITIES = {
+  SACRAMENTO: 'SACRAMENTO',
```
Comment (pmuchowski) [inline src/main/webapp/app/config/constants.ts]: as above

File: src/main/webapp/app/shared/layout/header/menus/sacramento.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Translate } from 'react-jhipster';
+import { NavLink as Link } from 'react-router-dom';
+import { NavItem, NavLink } from 'reactstrap';
+
+export const SacramentoMenu = props => (
+  // tslint:disable-next-line:jsx-self-close
```
Comment (pmuchowski) [inline src/main/webapp/app/shared/layout/header/menus/sacramento.tsx]: I don't think this is needed

File: src/main/webapp/app/modules/login/login.tsx (hunk 1 above)
Comment (jkondrat) [reply src/main/webapp/app/modules/login/login.tsx]: it's 'route'

File: src/main/java/org/benetech/servicenet/security/AuthoritiesConstants.java (hunk 2 above)
Comment (jkondrat) [reply src/main/java/org/benetech/servicenet/security/AuthoritiesConstants.java]: done

**Pull request merged**